### PR TITLE
Uwp fixes

### PR DESCRIPTION
**Pull request by gentledepp**
Hi!
Second attempt.
I forked @maxim-tkachenko s repository with uwp support and kind of finished it. If I am missing anything, let me know.

- created new nuspecs with updated dependencies and also .symbols. packages
- added xamarin.forms sample projects demonstating its use with ios,android and uwp
- added the "OneWayToSource" property "IsBlank" on the forms SignaturePadView so buttons can be disabled when there is no signature (see xamarin.forms sample for a demonstration)
- also, on UWP, loading and saving points is supported (special thanks to @gtbuchanan)

### CODE REVIEW

File: src/SignaturePad.UWP/SignaturePad.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+        #endregion
+
+        public void LoadPoints(Point[] loadedPoints)
+        {
+            if (loadedPoints == null || loadedPoints.Count() == 0)
+                return;
+
+            var wasBlank = IsBlank;
+
+            // unfortunately UWP does not allow to create InkStrokes from code..... crap...
```
Comment (gtbuchanan) [inline src/SignaturePad.UWP/SignaturePad.xaml.cs]: Your statement is incorrect. You can create an `InkStroke` on UWP with `InkStrokeBuilder`.

    var strokeBuilder = new InkStrokeBuilder();
    var stroke = strokeBuilder.CreateStrokeFromInkPoints(loadedPoints.Select(p => new InkPoint(p, 1f)), Matrix3x2.Identity);
    InkCanvas.InkPresenter.StrokeContainer.AddStroke(stroke);

Comment (gentledepp) [inline src/SignaturePad.UWP/SignaturePad.xaml.cs]: AWESOME!!!!!
Ok then I will implement that too.
Thanks man - I have litterally been searching for this piece of API for half an hour (unpaid of course...)

Comment (gtbuchanan) [inline src/SignaturePad.UWP/SignaturePad.xaml.cs]: No problem. I just updated the code to support the strokes being separated by Point(0, 0).

**Comment by gentledepp** [comment]
@gtbuchanan: I added your "LoadPoints" support and edited it a bit (for loading with the correct color and stroke thickness)
it works like a charm.
So nothing else open, I guess 👍

**Comment by gentledepp** [comment]
I just ran my Xamarin.Forms sample and noticed that all the features work on UWP and iOS now,
however, when I want to save the signature as a png on Android, I always end up with an empty picture.

Spoiler alert: I did change nothing at this part of the code. Do you have any idea what the problem could be??

**Comment by gtbuchanan** [comment]
@gentledepp It wasn't from your changes. I'd consider it an existing bug. For Android, you have to set the image stream position back to 0 before you read from it. Example:

    var image = await SignaturePad.GetImageStreamAsync(SignatureImageFormat.Png);
    image.Position = 0;
    ViewModel.SubmitCommand?.Execute(image);

**Comment by gentledepp** [comment]
@gtbuchanan: great! fixed that as well.

now one more thing:
I noticed that, when you call "LoadPoints" the current strokes are not removed on any platform. Does that make sense?

For me it would make more sense, if it would clear the canvas and load the points thereafter.
What do you guys think?

**Comment by gtbuchanan** [comment]
@gentledepp That would be a breaking change, so I would avoid doing that. Some people might want that functionality. If you need to clear it before adding new points, I would just expose a "Clear" method you can call beforehand.

**Comment by gentledepp** [comment]
sure you're right.
So I guess this is ready for merge then as the "clear" method would be another feature.

**Comment by gtbuchanan** [comment]
@gentledepp, looks like @mattleibow implemented most of this in the latest release.

**Comment by mattleibow** [comment]
I implemented most of UWP in the last release (v1.5), but I am doing a total re-write as well. This is to clean up the codebase to improve consistency across platforms/APIs as well as to have more code re-use.

The APIs are almost the same, but I have added some more overloads and features. I have also "split" the signature pad view and the actual drawing surface. This is useful for if you want a signature surface, but do no want the labels and lines. 

I hope to release a beta soon.

**Pull request closed without merging**